contractcourt/mock_registry_test.go

```diff
@@ -29,6 +29,11 @@ func (r *mockRegistry) NotifyExitHopHtlc(payHash lntypes.Hash,
 	wireCustomRecords lnwire.CustomRecords,
 	payload invoices.Payload) (invoices.HtlcResolution, error) {
 
+	// Exit early if the notification channel is nil.
+	if hodlChan == nil {
+		return r.notifyResolution, r.notifyErr
+	}
+
```

Comment:
Q: you mean `notifyChan ` is nil ? Seems like its always initialized via `newIncomingResolverTestContext` ?

Reply:
hmmm nope it's `holdChan`, moved this change to the relevant commit to make it more clear.